api/app/reviews/api.py

```diff
@@ -396,11 +416,11 @@ def get(self):
         limit = args['limit']
         sort_column = args['sort_column']
 
-        reviewer = db.session.query(AppUser).get(user_id).is_reviewer(event_id)
+        reviewer = review_repository.get_reviewer(event_id, user_id)

```

Comment:
Rather use user_repository to get the user and call is_reviewer on the result of that instead (see similar comment in repository)